### DISCUSSION TITLE
New Source: Railsr (Formerly railsbank)

### DISCUSSION THREAD

**Discussion by talShtark** [airbytehq/airbyte#36102] [New Connector Request]
## Tell us about the new connector you’d like to have
We would like to have a source connector for [Railsr](https://www.railsr.com/) REST [API](https://docs.railsr.com/docs/api/5ff00bb1936ca-railsbank-api).

## Describe the context around this new connector
Railsr is a Banking-as-a-service (BaaS) platform.
It enables customers to provide financial services to their clients (Railsr endusers).

With Railsr, We can issue credit/debit cards, manage financial ledgers, and make transactions.
A simple way to move our client's financial data with a simple mechanism to central storage will enable us to create meaningful analytics that affects business decisions.

Railsr has a simple REST API. Users should authenticate with an "API-Key" header.
With this API, Railsr customers can fetch data like:
GET /cards
GET /transactions
GET /endusers
etc...

Pagination can be done with the the`items_per_page` / `offset` parameters and incremental fetching with the `from_date ` and `to_date` parameters.

## Describe the alternative you are considering or using
Without this connector, we would have to build our mechanism to fetch the relevant data.

**Comment by marcosmarxm** [comment]
Thanks @talShtark are you interested in building this connector yourself?

**Comment by talShtark** [comment]
@marcosmarxm 
I would be happy to try out developing the integration on my own

[a comment by an automated account is not shown]
